### PR TITLE
[4.x] PHP 8 Support

### PR DESCRIPTION
**Pull request by driesvints**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -7,41 +7,42 @@ on:
...
 
   windows_tests:
 
     runs-on: windows-latest
     strategy:
       fail-fast: true
       matrix:
-        php: [7.3, 7.4]
+#        php: [7.3, 7.4, ^8.0]
+        php: [^8.0]
```
Comment (GrahamCampbell) [inline .github/workflows/tests.yml:45]: carrot is a typo i assume?

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (driesvints) [reply .github/workflows/tests.yml]: yep thanks

**Comment by nuernbergerA** [comment]
Hey @driesvints 
#170 should fix it.

**Comment by driesvints** [comment]
@taylorotwell Although that this library doesn't explicitly requires it, I've added `ext-fileinfo` to its dependencies which will allow us to catch the snag from above in the future more easily.

**Comment by crynobone** [comment]
I don't think there is a need for that. From what I see the ubuntu inage has php installed with fileinfo by default while windows doesn't seem to have it. 

I seen this issue all all tests with Laravel 9 at the moment (when running on windows CI)

**Comment by driesvints** [comment]
Okay I've removed it again. We'll see how it goes.

**Pull request merged**